pkg/utils/webhook/utils.go

```diff
@@ -0,0 +1,268 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
```

Comment:
are all of these functions used?
e.g. `DeserializeCommandline` or all the ensure functions?

Reply:
Ah, no, you are right. I wanted to go over this and remove unused functions, but forgot about it.